packages/shared/clickhouse/migrations/0004_scores_add_queue_id.down.sql

```diff
@@ -0,0 +1,2 @@
+ALTER TABLE scores
+DROP COLUMN queue_id;
```

Comment:
Covered via https://github.com/langfuse/langfuse/pull/3960/files#diff-ac64ef43424291171fd4ebfbd3fbaca280f563f6ddce31e08b834586804dea34R15. Can be removed.

Reply:
Thanks!